lib/utils/date_time_util.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:intl/intl.dart';
+
+class DateTimeUtils {
+  static String formatCurrentDate() {
+    final now = DateTime.now();
+    return DateFormat('EEEE, MMMM dd').format(now).toUpperCase();
+  }
+}
```

Comment:
Should we make this util more general by accepting any `DateTime` instance? Something like `static String formatFullDate(DateTime date) { ... }` 🙏 

Another option is creating an extension instead of introducing a new type. We can either:
- extend `DateFormat`:
  ```Dart 
  extension DateFormatExtension on DateFormat {
    static DateFormat fullDate() => DateFormat('EEEE, MMMM dd');
  }
  final formatted = DateFormatExtension.fullDate().format(date);
  ```
- extend `DateTime`:
  ```Dart
  extension on DateTime {
    String get fullDate => DateFormat('EEEE, MMMM dd').format(this);
  }
  final formatted = date.fullDate;
  ```
What do you think?

Reply:
Thank you 🙏 , creating an extension on DateTime is better than introducing a new type. I updated it at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/56/commits/6f086676d7d1ac72169627bae34a0f5849c7f28a